FLYWAY_MIGRATION.md

```
@@ -0,0 +1,39 @@
+# Flyway Migration Guide
+
+## Initial Setup Procedure
+
+### Existing DB (production / development server)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)